apps/vctags/Tags.v3

```diff
@@ -50,6 +53,17 @@ class Tag {
 
 		return id.toString();
 	}
+	// functions to escape '/' and '$' characters
+	private def escapeChars(s: string) -> string {
+		return Arrays.foldR(Arrays.map(s, escapeChar), Strings.concat, "");
```

Comment:
That's pretty inefficient. What about using a StringBuilder and a for loop?

Reply:
Done